docs/Video Streaming for Navigation Apps/Supporting Haptic Input/index.md

```diff
@@ -10,12 +10,7 @@ You will also need to implement [touch input support](Video Streaming for Naviga
 ## Automatic Focusable Rects
 SDL has support for automatically detecting focusable views within your UI and sending that data to the head unit. You will still need to tell SDL when your UI changes so that it can re-scan and detect the views to be sent.
 
-@![iOS]
```

Comment:
I think the iOS platform tag still needs to be here. The ending platform tag was not removed and is rendered on the site.